truckersmp-cli

```diff
@@ -784,6 +807,7 @@ When using standard Wine you should start the windows version of Steam first.
               Fallback: ./truckersmp]""")
     ap.add_argument(
       "-n", "--account", metavar="NAME", type=str,
+      default=get_current_steam_user(),
```

Comment:
I would not use `default=get_current_steam_user()`, and instead would call `get_current_steam_user()` in `check_args_errors()`:

```python
    # checks for updating
    if args.update and not args.account:
        args.account = get_current_steam_user()
        if not args.account:
            logging.info("Unable to find logged in user automatically.")
            sys.exit("Need the steam account name (-n name) to update.")
```